mne/epochs.py

```diff
@@ -3679,8 +3679,12 @@ def make_fixed_length_epochs(raw, duration=1., preload=False,
     ----------
     raw : instance of Raw
         Raw data to divide into segments.
+    id : int
+        The id to use (default 1).
```

Comment:
add version added

Reply:
Fixed. I made a mistake and placed it under the wrong argument.